### PR TITLE
Improve aliasing

### LINKED ISSUE #1861: ORA-00918: column ambiguously defined when selecting two columns that differ only in case (on Oracle)
This is almost the same issue as #1445 - which was fixed.

The fix did not handle the case where there are two columns that have the same name except with different case (e.g. `Id` and `id`).

### PR DESCRIPTION
**Pull request by timowest**
Fixes #1861

### CODE REVIEW

File: querydsl-sql/src/main/java/com/querydsl/sql/SQLSerializer.java [hunk 1]
```diff
@@ -176,7 +176,7 @@ public final SQLSerializer handleSelect(final String sep, final List<? extends E
             for (Expression<?> expr : expressions) {
                 if (expr instanceof Path) {
                     String name = ColumnMetadata.getName((Path<?>) expr);
-                    if (!names.add(name)) {
+                    if (!names.add(name.toLowerCase())) {
```
Comment (Shredder121) [inline querydsl-sql/src/main/java/com/querydsl/sql/SQLSerializer.java]: Locale dependent :(

**Comment by Shredder121** [comment]
We still need a configurable Locale or something actually.

File: querydsl-sql/src/main/java/com/querydsl/sql/SQLSerializer.java (hunk 1 above)
Comment (timowest) [inline querydsl-sql/src/main/java/com/querydsl/sql/SQLSerializer.java]: True, shall I use english until we have set up the Querydsl locale?

Comment (Shredder121) [inline querydsl-sql/src/main/java/com/querydsl/sql/SQLSerializer.java]: I think that's at least good.

**Comment by Shredder121** [comment]
Well, at least it looks fine to me.
And the build succeeds. :wink:

**Pull request merged**